showcases/vanilla-showcase/package.json

```
@@ -15,7 +15,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "file:./../../packages/foundations"
+    "@db-ui/foundations": "file:../../packagesi/foundations"
```

@nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations"
```